package.json

```diff
@@ -318,6 +322,11 @@
                 "key": "alt+Enter",
                 "when": "editorTextFocus && editorLangId == julia"
             },
+            {
+                "command": "language-julia.interrupt",
+                "key": "Ctrl+Shift+C",
```

Comment:
This technically overwrites a default kebinding (`Open New External Terminal`), but that doesn't seem like a big deal to me.

Reply:
Hm, don't we have the other keyboard shortcut already? I think overwriting default keybindings is probably something we should avoid...

Reply:
Yes, but it's very convenient not to have to focus the REPL to interrupt code, imho. Either is fine though, so I can remove this.

Reply:
I kind of feel we need a generic solution for this... Here is one idea: we could have a setting that says "enable conflicting julia keybindings", and that toggles one of these context values, and then we could use that in `when` clauses for all the keybindings that we want to introduce but that conflict with existing keybindings. The default for the setting would be `false`, but it would make it very easy for a user to toggle that setting, and then they would automatically get a large number of "better" keybindings, at the cost of them overwriting existing ones.

Reply:
Yes, I like that option. I wanted to rework our keybindings anyways, so I can implement that option while I'm at it. Ok to do that in a follow up?